### PR TITLE
Roslyn Shim - Round 2

### PR DESCRIPTION
**Pull request by TIHan**
Consuming the shim built in Roslyn, here -> dotnet/roslyn#35591 . That PR will actually remove most IVTs.

Still work in progress but want to make a PR to show the progress.

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -58,6 +58,7 @@
       https://vside.myget.org/F/vs-impl/api/v3/index.json;
       https://myget.org/F/roslyn_concord/api/v3/index.json;
       https://vside.myget.org/F/devcore/api/v3/index.json;
+      C:\roslyn\artifacts\packages\Debug\Release;
```
Comment (forki) [inline eng/Versions.props]: This doesn't look correct

Comment (TIHan) [reply eng/Versions.props]: It isn't and it will go away when the PR is ready. I did the exact same thing in the Round 1 shim PR.

Comment (forki) [reply eng/Versions.props]: Thx

File: vsintegration/src/FSharp.Editor/BlockComment/CommentSelectionService.fs [hunk 2]
```diff
@@ -1,19 +0,0 @@
-﻿namespace Microsoft.VisualStudio.FSharp.Editor
-
-open Microsoft.CodeAnalysis.CommentSelection
-open Microsoft.CodeAnalysis.Host.Mef
-open System.Composition
-open System.Threading.Tasks
-
-[<Shared>]
-[<ExportLanguageService(typeof<ICommentSelectionService>, FSharpConstants.FSharpLanguageName)>]
-type CommentSelectionService() =
```
Comment (TIHan) [inline vsintegration/src/FSharp.Editor/BlockComment/CommentSelectionService.fs]: This was deleted because I moved into Roslyn directly.

**Comment by TIHan** [comment]
This is 99% done. Do not merge this in until Roslyn has merged theirs first. 

Still having some trouble getting DocumentDiagnosticAnalyzers to kick off. Once those are working and do some testing, this will be 100% ready.

**Comment by TIHan** [comment]
Waiting for Roslyn's preview 4 branch. As soon as they have it, we update the package one more time. When Roslyn is ready to do an insertion, we merge and update the insertion with our bits.
// cc @JoeRobich

**Comment by brettfo** [comment]
I spoke with @333fred and he pointed me to the Roslyn insertion into VS.  I've updated `RoslynPackageVersion.txt` to match so as soon as this is green we'll merge it and start the official signed build then update the coordinated insertion.

**Comment by brettfo** [comment]
Awaiting signed build: https://dev.azure.com/devdiv/DevDiv/_build/results?buildId=2763846

**Pull request merged**